java/statement/builder/StatementTypeBuilder.java

```diff
@@ -192,7 +191,8 @@ default StatementType relates(Statement type) {
      */
     @CheckReturnValue
     default StatementType relates(String roleType, @Nullable String superRoleType) {
```

Comment:
Should we also overload this?

Reply:
yeah lets stick to that style, I had not overloaded it to force me to update all the users properly

Reply:
lets unify the other methods too

Reply:
and by 'unify' you mean overload other methods too, right? and remove `@Nullable` annotation?